src/leiningen/monolith.clj

```diff
@@ -181,13 +183,61 @@
   (checkouts/unlink project))
 
 
+;; ## Fingerprinting
+
+;; Leiningen's help framework doesn't officially support "subsubtasks" so
+;; we'll try our best to make it work by making them look like subtasks.
+
+(defn changed
+  "Show information about the projects that have changed since last :refresh.
+
+  Optionally takes one or more marker ids, or project selectors, to narrow the
+  information.
+
+  Usage:
+  lein monolith changed [project-selectors] [marker1 marker2 ...]"
```

Comment:
This should probably take `:bare` and spit out just the list of changed projects, but then passing zero markers or two or more markers wouldn't make much sense.

Reply:
Leaving this as "future improvement" since I think currently being able to emit a pretty report describing all the markers is nice to have, and there isn't an obvious `:bare` counterpart.